camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeper.java

```diff
@@ -280,20 +285,10 @@ private void runCollectorForTopicDir(FileSystem fs, String topic, Path topicSour
 
     List<Properties> jobPropsList = planner.createSweeperJobProps(topic, topicSourceDir, topicDestDir, fs);
 
-    for (Properties jobProps : jobPropsList) {
+    for (Properties jobProps : jobPropsList){
       tasksToComplete.add(runCollector(jobProps, topic));
```

Comment:
Lets leave as this - I just reverted to original version. Refactoring `camus` (which is a discontinued project) would be a time waste.